src/lightning_app/cli/cmd_clusters.py

```diff
@@ -307,21 +307,24 @@ def _cluster_status_long(cluster: V1GetClusterResponse, desired_state: V1Cluster
     duration = _format_elapsed_seconds(elapsed)
 
     if current_state == V1ClusterState.FAILED:
-        return dedent(
-            f"""\
-            The requested cluster operation for cluster {cluster_id} has errors:
-            {current_reason}
+        if not _is_retryable_error(current_reason):
+            return dedent(
+                f"""\
+                The requested cluster operation for cluster {cluster_id} has errors:
 
-            ---
-            We are automatically retrying, and an automated alert has been created
+                {current_reason}
 
-            WARNING: Any non-deleted cluster may be using resources.
-            To avoid incuring cost on your cloud provider, delete the cluster using the following command:
-                lightning delete cluster {cluster_id}
+                --------------------------------------------------------------
 
-            Contact support@lightning.ai for additional help
-            """
-        )
+                We are automatically retrying, and an automated alert has been created
```

Comment:
Not very clear what the alert does.

Reply:
This existed before this PR so I guess we can leave for a follow-up 😃